tests/rptest/scale_tests/many_partitions_test.py

```diff
@@ -589,13 +610,17 @@ def _tiered_storage_warmup(self, scale, topic_name):
         number of segments.
         """
 
-        warmup_segment_size = 32 * 1024
-        warmup_message_size = 32 * 1024
+        warmup_segment_size = scale.segment_size
+        # Because segments are rolled after a write we need to size messages
+        # such that actual segment size is as close as possible to the desired
+        # size. In default configuration we run with `log_segment_size_jitter_percent=5`
+        # adjust the messages size to always be larger than that.
+        warmup_message_size = math.ceil(scale.segment_size * 0.06)
```

Comment:
Can you explain this a bit more. 

Also you mention `log_segment_size_jitter_percent=5` but you removed the config for that above. Is that just the default in code?